### PR TITLE
Adds Configuration to skipp html tags

### PR DESCRIPTION
**Pull request by AlexejKossmann**
This PR allows to add html tags to configuration files, to be completly skipped from hyphernating.

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{
+    const ROOT_ID = 'huh_hyphenator';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root(static::ROOT_ID);
```
Comment (koertho) [inline src/DependencyInjection/Configuration.php]: Please skip the deprecation warning here with supporting the changes from symfony 4.2 as already done in utils bundle: https://github.com/heimrichhannot/contao-utils-bundle/blob/master/src/DependencyInjection/Configuration.php#L26

File: src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        $rootNode
+            ->children()
+                ->arrayNode('skip_tags')
```
Comment (koertho) [inline src/DependencyInjection/Configuration.php:35]: I think we should set script as a default value here?

File: src/Hyphenator/FrontendHyphenator.php [hunk 3]
```diff
@@ -108,10 +108,47 @@ function (HtmlPageCrawler $node, $i) use ($h, &$cache, $cacheEnabled, $objPage)
                 $html = $this->handleLineBreakExceptions($html, $objPage);
 
                 if (!$this->isHyphenationDisabled($objPage, Config::get('hyphenator_skipPages'))) {
+                    // mask tags configured to be skipped
+                    $skipTagCache = [];
+                    $skipTagCacheIndex = 0;
+
+                    foreach ($this->container->getParameter('huh_hyphenator')['skip_tags'] as $tag) {
+                        if (\in_array($tag, $this->voidElements)) {
```
Comment (koertho) [inline src/Hyphenator/FrontendHyphenator.php:116]: Can you check if the voidElement replacement in line 171 is still necessary?

**Review by koertho — CHANGES_REQUESTED** [review]
Looks good for me. I added just some little remarks. BUT :fearful: there is a test for this class and such a functionality should really have tests. So I would prefer to update the tests (and please move them to github actions).

File: src/Hyphenator/FrontendHyphenator.php (hunk 3 above)
Comment (AlexejKossmann) [reply src/Hyphenator/FrontendHyphenator.php:116]: yes, this is necessary, bcuz the regex differes on tags with closing element and without

**Review by koertho — APPROVED** [review]
Looks good for me!

**Pull request merged**